src/features/pages/404/content/not-found.module.scss

```diff
@@ -0,0 +1,16 @@
+@use 'src/features/styles/theme/theme-mixins' as *;
+@import 'src/features/styles/theme/variables';
+
+.not {
+  &_found {
+    padding: 16rem 0;
```

Comment:
we should remove this, let's have a call for this, you can handle it with `padding_block` prop, but it gives you `8rem` limit. so let's chat. :pray:

Reply:
@mohsen-deriv can we reach our needs via this approach?
https://github.com/binary-com/deriv-com/pull/4445/commits/3dc3acdae81f3e2389f2cce01e7c4c8e178b81a7